stglib/vec/dat2cdf.py

```diff
@@ -34,21 +36,19 @@ def dat_to_cdf(metadata):
 
     ds = aqdutils.check_attrs(ds, inst_type="VEC")
 
-    dsvhd = load_vhd(basefile)
+    dssen = load_sen(basefile, metadata)
```

Comment:
If we already have a Dataset with the metadata, I prefer to use that instead of the metadata dict.